specification.md

```diff
@@ -0,0 +1,152 @@
+# Specification
+
+This specification provides a mechanism for projects to report information about their security in a machine-processable way. It is formatted as a YAML file to make it easy to read and edit by humans.
+
+Values that are included within the specification may be required or optional. Optional values are reccommendations from the Open Source Security Foundation's _Identifying Security Threats Working Group_, but may not be prudent for all use cases.
+
+The following values are part of Security Insights Specification `v1.0.0`
+
+## Header
+Here is an example of the header section, which holds high-level information about the project and its security documentation.
+
+```yaml
+header:
+  schema-version: 1.0.0
+  last-updated: '2021-09-01'
+  last-reviewed: '2022-09-01'
+  expiration-date: '2023-08-31T10:10:09.000Z'
+  project-url: https://github.com/foo/bar
+  commit-hash: 4dbf78ebc006ee5f668c0a74876ef8d6db9485be
+  project-release: '1.2.0'
+  changelog: https://github.com/foo/changelog.md
+  license: https://git.foo/license
+  parent-security-yaml: https://blah.com/ossf-security.yaml
+```
+
+Now let’s break that down:
```

Comment:
A table format would be a nice alternative to a list.

Reply:
I see the value in this argument... will take a stab at this in a bit to see how it looks.